kimhaejung/.env.sample

```
@@ -0,0 +1,10 @@
+DB_CONNECTION = mysql
+DB_HOST = HOST
+DB_USERNAME = USERNAME
+DB_PASSWORD = PASSWORD
+DB_DATABASE = DATABASE
+DB_PORT = DB_PORT
+DB_LOGGING = BOOLEAN
+
+DATABASE_URL= "mysql://username:password@host:port/dbName"
+port = port
```

다른 환경변수와 동일하게 통일성을 맞출 수 있도록 수정해주세요!

```suggestion
PORT = PORT
```